client/src/helpers/nova/hooks/useGenerateSlotsTable.ts

```diff
@@ -140,7 +140,7 @@ function getSlotCommitmentTableRow(
                 tableData = {
                     type: TableCellType.Stats,
                     data: slotIndex.toString(),
-                    href: `/${network}/slot/${slotIndex}?tab=Txs`,
+                    href: `/${network}/slot/${slotIndex}`,
```

Comment:
the line above with `?tab=RefBlocks` should also be removed, there are no tabs in the slot page